examples/api/src/index.ts

```diff
@@ -113,12 +114,34 @@ async function asyncMain() {
     }
   ]
 
+  if (!process.env.MAILGUN_API_KEY) throw new Error('No MAILGUN_API_KEY defined in environment.')
+  if (!process.env.MAILGUN_BASE_URL) throw new Error('No MAILGUN_BASE_URL defined in environment.')
+  if (!process.env.MAILGUN_WEBHOOK_SECRET)
+    throw new Error('No MAILGUN_WEBHOOK_SECRET defined in environment.')
+
+  let mailProvider
+  if (
```

Comment:
I guess there is no need for the if condition over here.

Reply:
Why is there no need for the if condition?